patches/components-sync-driver-BUILD.gn.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/components/sync/driver/BUILD.gn b/components/sync/driver/BUILD.gn
+index 548c712f886934b34ab0ec3e7feca09cde229d1a..807db31e4fc78ad6bbd710d69c2ec74123f7be5f 100644
+--- a/components/sync/driver/BUILD.gn
++++ b/components/sync/driver/BUILD.gn
+@@ -133,6 +133,8 @@ jumbo_static_library("driver") {
+   defines = [ "SYNC_USER_AGENT_PRODUCT=$sync_user_agent_product" ]
+ 
+   configs += [ "//build/config/compiler:wexit_time_destructors" ]
++
```

Comment:
don't add extra newlines in patches

Reply:
fixed in https://github.com/brave/brave-core/pull/5294/commits/b2fdf8c10dd8d42e0f3add64c775cfd12b8abd57